### PR TITLE
Improvement/new guidelines contribution

### PR DESCRIPTION
**Pull request by mmexvr**
**Describe pull-request**  
_Describe what the pull-request is about_
Adding contribution guidelines

Add comments on the lines that need to be changed

**DO NOT merge**

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #579

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+  Corporate UI is a toolbox with **UI components** that makes it easier to use reusable and modular components for your application. For example Header or a footer are two components, but also buttons and dropdowns are also included.
+
+  Scania-Theme is a packages to contains the Scania specific styling for everyone of those components in Corporate UI toolbox. Every component has a unique styling with Scania look and feel.
+
+
+## Components
+
+ Header,footer or a button are all components in our Design System, the difference between them is how they technically are made.
+
+ `<c-header></c-header>` is a web component, while a `<button> Click Me </button>` is a regular HTML element that already exist in the HTML Standard. One thing easy thing to look at is the dash, for example "c-header", that indicates it is a web-component.
```
Comment (helloimela) [inline contribution.md:14]: "One easy thing to differentiate between those is by looking at the dash in the element name, for example...."

File: contribution.md [hunk 2]
```diff
@@ -0,0 +1,500 @@
+# Developing a component
+
+  This is a tutorial how to setup a project with Corporate-UI components with Scania-theme styling.
+
+  Corporate UI is a toolbox with **UI components** that makes it easier to use reusable and modular components for your application. For example Header or a footer are two components, but also buttons and dropdowns are also included.
```
Comment (helloimela) [inline contribution.md:5]: "For examples, header, footer, buttons, and dropdowns"

File: contribution.md [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+
+  Scania-Theme is a packages to contains the Scania specific styling for everyone of those components in Corporate UI toolbox. Every component has a unique styling with Scania look and feel.
```
Comment (helloimela) [inline contribution.md:7]: "Scania-theme is a package that contains Scania specific styling...."

File: contribution.md [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+
+
+## Components
+
+ Header,footer or a button are all components in our Design System, the difference between them is how they technically are made.
```
Comment (helloimela) [inline contribution.md:12]: ...the difference between them is how they technically made

File: contribution.md [hunk 5]
```diff
@@ -0,0 +1,500 @@
...
+
+ `<c-header></c-header>` is a web component, while a `<button> Click Me </button>` is a regular HTML element that already exist in the HTML Standard. One thing easy thing to look at is the dash, for example "c-header", that indicates it is a web-component.
+
+ A web component can be used in the same way as a regular element in the HTML standard. Difference is what is under the hood. You can read more about it here https://developer.mozilla.org/en-US/docs/Web/Web_Components
+
+ All web-components in Scania digital design system have namespace with  `<c-nameOfComponent>`
+
+## Audience
+
+This is for developers working with web application and have knowledge about web development. Components are avaliable for **react**,**angular** and **vue**
```
Comment (helloimela) [inline contribution.md:22]: Components are available for ....

File: contribution.md [hunk 6]
```diff
@@ -0,0 +1,500 @@
...
+
+We have setup a few example repositories for react, angular and vue
+ - [React]()
+ - [Angular]()
+ - [Vue]()
+
+
+ ## Stencil
+
+ To be able to create a component in Corporate-Ui you need to understand what tool we are using to make it happened. We are using a **Web Component compiler** called [StencilJS](https://stenciljs.com/docs/introduction). This is the core for our components and how we build them. You can more in detail in their documentation.
```
Comment (helloimela) [inline contribution.md:32]: You can read more in Stencil documentation

File: contribution.md (hunk 1 above)
Comment (mmexvr) [reply contribution.md:14]: Added

**Comment by mmexvr** [comment]
added in sdds

**Pull request closed without merging**